addon/-private/properties/is-present.js

```diff
@@ -1,31 +1,37 @@
 import { findElement } from 'ember-cli-page-object';
 
 /**
- * Validates if an element or set of elements are in the DOM. In most cases,
- * it is preferred to use `isVisible` instead as it provides a slightly
- * stronger guarantee. However, a use case for `exists` is to check the
- * presence of a <meta> tag on the page.
+ * Validates if any element matching the target selector is rendered in the DOM.
+ *
+ * `isPresent` vs. `isVisible`:
+ *   - Both validate that an element matching the target selector can be found in the DOM
+ *   - `isVisible` additionally validates that all matching elements are visible
+ *
+ * Some uses cases for `isPresent` over `isVisible`:
+ *   - To check for the presence of a tag that is never visible in the DOM (e.g., <meta>).
+ *   - To validate that, even though an element may not currently be visible, it is still in the DOM.
+ *   - To validate that an element has not merely been hidden but has in fact been removed from the DOM.
```

Comment:
I really liked the idea of calling out the difference between `isVisible` and `isPresent` and wanted to add a bit more detail to the explanation.

Reply:
Totally agreed!